src/common/components/Solana/TokenRegistry.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import {
+  TokenListProvider,
+  TokenInfoMap,
+  TokenInfo,
+  TokenListContainer,
+  Strategy,
+} from '@solana/spl-token-registry';
+
+const TokenRegistryContext = React.createContext<TokenInfoMap>(new Map());
+
+type ProviderProps = { children: React.ReactNode };
+
+export function TokenRegistryProvider({
+  children,
+}: ProviderProps): JSX.Element {
+  const [tokenRegistry, setTokenRegistry] = React.useState<TokenInfoMap>(
+    new Map()
+  );
+
+  React.useEffect(() => {
+    new TokenListProvider()
+      .resolve(Strategy.Solana)
+      .then((tokens: TokenListContainer) => {
+        const tokenList = tokens.filterByClusterSlug('devnet').getList();
```

Comment:
Instead of hardcoding 'devnet', can we use a global config file where we define a network const that is used here and in WalletProviders.tsx to set the network?

Im worried that in the future we may forget to update one of the networks.

Reply:
done env var

Reply:
separate file finally